exporter/otlphttpexporter/otlp.go

```diff
@@ -80,7 +82,15 @@ func (e *exporterImp) pushTraceData(ctx context.Context, traces pdata.Traces) (i
 		return traces.SpanCount(), consumererror.Permanent(err)
 	}
 
-	err = e.export(ctx, e.tracesURL, request)
+	const TOKEN = "$TOKEN"
```

Comment:
I would suggest to put this constant into a more visible place, like even the section around line 51. Also, constant names are typically not all-caps. I think a more descriptive name could help as well